lib/mbta_v3_api/store.ex

```diff
@@ -25,11 +26,8 @@ defmodule MBTAV3API.Store do
 
   @doc """
   Retrieve all records that match the given filter keys.
+  When given a single keyword list, all keys in the list must match.
+  When given a list of keyword lists, must match any of the keyword lists
   """
-  @callback fetch(keyword()) :: [JsonApi.Object.t()]
-
-  @doc """
-  Retrieve all records that match any of the given filter keyword sets
-  """
-  @callback fetch_any([keyword()]) :: [JsonApi.Object.t()]
+  @callback fetch(fetch_keys()) :: [JsonApi.Object.t()]
```

Comment:
collapsed these into 1 function as recommended in https://github.com/mbta/mobile_app_backend/pull/190